make/ex_doc.exs

```diff
@@ -151,7 +151,7 @@ current_datetime = System.os_time() |> DateTime.from_unix!(:native)
 config = [
   proglang: :erlang,
   source_url_pattern: source_url_pattern,
-  assets: Path.join(cwd, "/assets"),
+  assets: %{ Path.join(cwd, "/assets") => "assets" },
```

Comment:
Next error I get from the nix build

```
warning: the VM is running with native name encoding of latin1 which may cause Elixir to malfunction as it expects utf8. Please ensure your locale is set to UTF-8 (which can be verified by running "locale" in your shell) or set the ELIXIR_ERL_OPTIONS="+fnu" environment variable
** (FunctionClauseError) no function clause matching in IO.chardata_to_string/1    
    
    The following arguments were given to IO.chardata_to_string/1:
    
        # 1
        %{"/build/source/lib/stdlib/doc/assets" => "assets"}
```

Reply:
stack trace

```
    (elixir 1.16.3) lib/io.ex:687: IO.chardata_to_string/1
    (elixir 1.16.3) lib/path.ex:662: Path.join/2
    (ex_doc 0.32.1) lib/ex_doc/formatter/html.ex:294: ExDoc.Formatter.HTML.copy_assets/2
    (ex_doc 0.32.1) lib/ex_doc/formatter/html.ex:288: ExDoc.Formatter.HTML.generate_assets/3
    (ex_doc 0.32.1) lib/ex_doc/formatter/html.ex:25: ExDoc.Formatter.HTML.run/3
    (ex_doc 0.32.1) lib/ex_doc/cli.ex:69: anonymous fn/6 in ExDoc.CLI.generate/3
    (elixir 1.16.3) lib/enum.ex:1700: Enum."-map/2-lists^map/1-1-"/2
    (elixir 1.16.3) lib/kernel/cli.ex:136: anonymous fn/3 in Kernel.CLI.exec_fun/2
```

Reply:
I guess this comes from https://github.com/elixir-lang/ex_doc/blob/v0.34.0/CHANGELOG.md#v0340-2024-05-30? i.e. I think this requires a specific version of ex_doc now. Unsure if that is intended?

Reply:
We will always require a specific version of ex_doc, which one is listed in https://github.com/erlang/otp/blob/master/make/ex_doc_vsn

Reply:
Ahh, okay! Let me update that and keep testing thanks!

Reply:
There are some bugs in v0.34.0 of ex_doc that cause some issues, so you may have to wait for v0.35.0 in order for everything to pass.